documentation/modules/appendix/pages/model-evolution.adoc

```diff
@@ -0,0 +1,232 @@
+////
+Copyright (c) 2023 Robert Bosch Manufacturing Solutions GmbH
+
+See the AUTHORS file(s) distributed with this work for additional information regarding authorship. 
+
+This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0.
+If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/
+SPDX-License-Identifier: MPL-2.0
+////
+
+:page-partial:
+
+[[model-evolution]]
+= Model Evolution
+
+[[why-is-model-evolution-needed]]
+== Why is Model Evolution needed?
+
+Seldomly, an Aspect model can be created completely in one step from ideation to conceptualization.
+
+Authoring an Aspect Model is comparable to the creation of a software artifact. At first, a domain
+model draft or concept map is created to capture the understanding of the respective domain experts
+(cf. https://en.wikipedia.org/wiki/Domain-driven_design[domain-driven design]), which is then
+formalized into an actual Aspect Model. After multiple iterations, each of which improves the
+modeled concepts and the overall quality of the model, a first "complete" version is created. This
+first version might contain the minimal subset of relevant Properties complete with correct
+Characteristics and proper descriptions.
+
+This "Version 1.0" of the model can then be used to implement the first version of the corresponding
+Aspect. Imagine that the service is running and delivers data that corresponds to the "Version 1.0"
+of the Aspect model. Clients are developed as well, possibly by different parties, and happily
+consume the Aspect data. Now the following happens: The domain experts find out that the model is
```

Comment:
I don't like the "domain experts finds out that the model is **not correct**". In the further paragraph is explained well, 
Maybe just something like "need for change". "not correct" sounds bad and means "okay, people shouldnt use the information, its not correct. It should be revoked and replaced."



"consume the Aspect data. However sooner or later there will be a need for change in the aspect model, be it 
missing relevant Properties, existing Properties could use wrong
Characteristics, descriptions or names, or are missing localizations. This can happen for many
reasons:"


Proposal (that I wrote before realizing it's already included in the further paragraph).
<<<
In a world of evolving digitalization this is unlikely to remain stable "forever". Additional use cases may require additional data belonging to an aspect, so additional properties may be required. Standards, legal requirements as well as identified shortcomings/bugs in an existing model are some other sources that can mandate changing an 
>>>